include/cute/util/print.hpp

```diff
@@ -70,13 +70,13 @@ print(char c) {
 CUTE_HOST_DEVICE
 void
 print(signed char a) {
-  printf("%hhd", a);
+  printf("%d", (int)(a));
```

Comment:
we do not use c style casts in our code. can you please convert these to static casts instead?

Reply:
done